usr/src/lib/libdhcpagent/common/dhcpagent_ipc.h

```diff
@@ -257,6 +260,8 @@ typedef hrtime_t dhcp_ipc_id_t;
  *	 are intentionally identical; code in dhcpagent_ipc.c counts on it!
  */
 
+#pragma pack(4)
```

Comment:
Can we add a comment here so that folks who come here understand why we we're packing this in the future?